server/app/blocks.go

```diff
@@ -124,6 +126,18 @@ func (a *App) DeleteBlock(c store.Container, blockID string, modifiedBy string)
 		return err
 	}
 
+	if block.Type == model.TypeImage {
+		fileName, fileIDExists := block.Fields["fileId"]
+		if fileName, fileIDIsString := fileName.(string); fileIDExists && fileIDIsString {
+			filePath := filepath.Join(block.WorkspaceID, block.RootID, fileName)
+			err = a.filesBackend.RemoveFile(filePath)
+
+			if err != nil {
+				return err
```

Comment:
I think we need to broadcast the block delete even if the image delete fails.

Reply:
Agreed. Good call. I've decided to just log the error and proceed.